SqlServer/Import-BacpacToSqlServer/Import-BacpacToSqlServer.psm1

```diff
@@ -134,23 +192,17 @@ function Import-BacpacToSqlServer
 
         if (!(Test-SqlServer $DataSource))
         {
-            Write-Warning ("Could not find SQL Server at `"$DataSource`"!")
+            Write-Warning "Could not find SQL Server at '$DataSource'!"
             $DataSource = 'localhost'
 
             if (!(Test-SqlServer $DataSource))
             {
-                throw ('Could not find any SQL Server instances!')
+                throw 'Could not find any SQL Server instances!'
             }
         }
 
 
-
-        # Checking DatabaseName validity and handling if there's already a database with this name on the target server.
-        if ([string]::IsNullOrEmpty($DatabaseName))
-        {
-            $DatabaseName = $bacpacFile.BaseName
-        }
-
+        # Handling the case when there's already a database with this name on the target server.
         $databaseExists = Test-SqlServerDatabase -SqlServerName $DataSource -DatabaseName $DatabaseName
```

Comment:
This doesn't actually work in docker, where we use SQL authentication. Adding the `$Username` and `Password` worked for me:

```powershell
        $securePassword = $null
        if (![string]::IsNullOrEmpty($Password))
        {
            $securePassword = ConvertTo-SecureString $Password -AsPlainText -Force
        }

        if (!(Test-SqlServer $DataSource $Username $securePassword))
        {
            Write-Warning "Could not find SQL Server at '$DataSource'!"
            $DataSource = 'localhost'

            if (!(Test-SqlServer $DataSource))
            {
                throw 'Could not find any SQL Server instances!'
            }
        }

        # Handling the case when there's already a database with this name on the target server.
        $databaseExists = Test-SqlServerDatabase -SqlServerName $DataSource -DatabaseName $DatabaseName -UserName $Username -Password $securePassword
```